http/context.go

```diff
@@ -0,0 +1,79 @@
+package http
+
+import (
+	"fmt"
+
+	"github.com/gookit/color"
+
+	"github.com/goravel/framework/contracts/config"
+	"github.com/goravel/framework/contracts/http"
+)
+
+type Driver string
+
+const (
+	DriverGin   Driver = "gin"
+	DriverFiber Driver = "fiber"
```

Comment:
S: Goravel doesn't need to know which the driver is, any driver that implements contact is ok.